samples/cellular/nrf_cloud_multi_service/overlay_coap.conf

```diff
@@ -61,6 +61,7 @@ CONFIG_NRF_CLOUD_ALERT=y
 CONFIG_NRF_CLOUD_LOG_DIRECT=y
 CONFIG_NRF_CLOUD_LOG_OUTPUT_LEVEL=3
 CONFIG_NRF_CLOUD_LOG_LOG_LEVEL_INF=y
+CONFIG_NRF_CLOUD_FOTA_SUPPORT=y
```

Comment:
I changed the Kconfig to `CONFIG_NRF_CLOUD_FOTA_POLL`.

I think it is better to only compile it when needed; otherwise this just adds to build time unnecessarily when it is not actually used.